p4include/v1model.p4

```diff
@@ -133,14 +145,17 @@ parser Parser<H, M>(packet_in b,
                     inout standard_metadata_t standard_metadata);
 control VerifyChecksum<H, M>(in H hdr,
                              inout M meta);
+@pipeline
```

Comment:
Previously, the skipControls policy receives the list of control blocks to skip from the hard-coded names in v1model. https://github.com/p4lang/p4c/blob/b25ba2dc23d147a9efdcc3ed39c49e72fc84ec34/backends/bmv2/midend.cpp#L126  I was looking for a way to generalize how skipControls (and other policies) computes the list of control blocks. The solution I came up with was to use special annotations in the architecture file. That way, at least I don't have to change the backend code for a different architecture. By putting the annotation on the type, only the person who designs the architecture needs to care about this detail, and it's not visible to the P4 programmer. It also affects the checksum passes, as I need a way to identify which control block is updateChecksum. It would be argument 5 in v1model or argument n ( n != 5) in PSA, It may not have the name `updateChecksum` in PSA, or other architecture that people come up with. I am not sure how to identify updateChecksum without using annotation?